src/components/organisms/ProfileModal/UserInformationContent/UserInformationContent.tsx

```diff
@@ -80,37 +80,37 @@ export const UserInformationContent: React.FunctionComponent<UserInformationCont
     }
   }, [user]);
 
-  const profileLinks = useMemo(() => {
-    const makeEditProfileLink = (profileLink?: ProfileLink) => () => {
-      setProfileLinkToEdit(profileLink);
-      setUserProfileMode(UserProfileMode.EDIT_PROFILE_LINK);
-    };
-
-    return (
-      <>
-        <ul className="UserInformationContent__profile-links">
-          {user?.profileLinks?.map((profileLink) => (
-            <li key={profileLink.title}>
-              {profileLink.title}{" "}
-              <button
-                className="button--a"
-                onClick={makeEditProfileLink(profileLink)}
-              >
-                Edit
-              </button>
-            </li>
-          ))}
-        </ul>
-
-        <button
-          className="UserInformationContent__add-profile-link button--a"
-          onClick={makeEditProfileLink(undefined)}
-        >
-          Add a profile link
-        </button>
-      </>
-    );
-  }, [setProfileLinkToEdit, setUserProfileMode, user?.profileLinks]);
+  // const profileLinks = useMemo(() => {
+  //   const makeEditProfileLink = (profileLink?: ProfileLink) => () => {
+  //     setProfileLinkToEdit(profileLink);
```

Comment:
This prop is no longer used, you should remove it and check if the code compiles successfully.

Reply:
Thank you a lot!
fixed in 1976e83f6373fa481d0d82d53a6e4bdd21a54408

Reply:
I fear that this actually does compile successfully even though the prop is unused. Which means that we should probably fix this in the near future.

Reply:
The code was fine being commented, what I meant is that `setProfileLinkToEdit` is no longer used anywhere but it's passed as  prop to the component.

Reply:
@denisdimitrov 
I see what you mean, I'm scaring to remove `setProfileLinkToEdit` and the code changes in `src/components/organisms/ProfileModal/ProfilePopoverContent.tsx` cause later it could create a lot of merge conflicts.
WDYT?

Reply:
Sure, it's going in github anyways, we can merge it like this.